ember-toucan-core/src/components/form/controls/input.ts

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+interface ToucanFormControlsInputComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    isDisabled?: boolean;
```

Comment:
If anyone has any advice as to why this is erroring: https://github.com/CrowdStrike/ember-toucan-core/actions/runs/4247201741/jobs/7384984044 would appreciate it!